pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -113,14 +112,6 @@ stdenv.mkDerivation rec {
   patches = [
     # patch to fix python-test
     ./darwin.patch
-    # in master post 8.0.0, see https://github.com/apache/arrow/pull/13182
-    (fetchpatch {
-      name = "fix-pkg-config.patch";
-      stripLen = 1;
-      excludes = [ "src/arrow/engine/arrow-substrait.pc.in" ]; # for < 8.0.0
-      url = "https://github.com/apache/arrow/commit/a242eb17362c0352fc3291213542c48abfe18669.patch";
```

Comment:
I don't get how this is not needed anymore.

Reply:
the package is at 9.0.0